woocommerce/woocommerce-ios
# [IPPInAppFeedback] Prepare for release

Closes: #8530


## Description
This PR switches the `.IPPInAppFeedbackBanner` feature flag, to enable In-Person Payments feedback survey banner project.

## Testing instructions
1. Switch the run scheme build configuration to `release-alpha`
2. On a US/CA store, with COD enabled:
3. Run the app, and confirm that the In-Person Payments feedback survey appears when navigating to the `Orders` tab.

If COD is not enabled when initially running the app, please enable it and re-run the app.

## Screenshots

<img width=450 src="https://user-images.githubusercontent.com/3812076/214539146-9a1c25ab-faf0-4a3d-a0f2-ab44bb2435ec.gif">


---
